mempool/reactor_test.go

```diff
@@ -614,7 +614,7 @@ func checkTxsInOrder(t *testing.T, txs types.Txs, reactor *Reactor, reactorIndex
 	// Check that all transactions in the mempool are in the same order as txs.
 	reapedTxs := reactor.mempool.ReapMaxTxs(len(txs))
 	require.Equal(t, len(txs), len(reapedTxs))
-	for i, tx := range txs {
+	for i, tx := range reapedTxs {
```

Comment:
I think this cannot stay this way because the check is reundant (compares `tx` which is assigned the value `reapedTx[i]` with `reapedTxs[i]`

Reply:
That's right. We actually don't need to check that txs are in order, just that they are in the mempool. I changed them.